docs/dev/connectors/kafka.md

```diff
@@ -413,7 +413,7 @@ stream.addSink(myProducer);
 {% highlight scala %}
 val stream: DataStream[String] = ...
 
-Properties properties = new Properties
+val properties = new Properties
```

Comment:
Thanks for the contribution @huzekang 
Could you also fix `docs/dev/connectors/kafka.zh.md` the same way before merging?